network/p2p/p2pnode/libp2pNode.go

```diff
@@ -133,12 +138,18 @@ func (n *Node) Stop() (chan struct{}, error) {
 	return done, nil
 }
 
-// AddPeer adds a peer to this node by adding it to this node's peerstore and connecting to it
+// AddPeer adds a peer to this node by adding it to this node's peerstore and connecting to it.
+// The following benign errors are expected during normal operations from libP2P:
+// 	- host fails to connect to peer
+// All errors returned from this function can be considered benign.
```

Comment:
Same clarification needed here about `benign errors`.

Reply:
@kc1116 if _all_ errors returned can be considered benign, then we may skip listing the individual errors, it makes the `godoc` more maintainable.